frontend/public/locales/en/apply-adult.json

```
@@ -35,6 +35,32 @@
       "date-of-birth-year-required": "Date of birth must include a year"
     }
   },
+  "marital-status": {
+    "page-title": "Marital status",
+    "marital-status": "What is your current marital status?",
+    "spouse-or-commonlaw": "Spouse or common-law partner information",
+    "provide-sin": "Provide the Social Insurance Number (SIN) of your current spouse or common-law partner. The information is required to calculate your adjusted family net income.",
+    "required-information": "Your spouse or common-law partner must submit their own application for the Canadian Dental Care Plan.",
+    "back-btn": "Back",
+    "continue-btn": "Continue",
+    "cancel-btn": "Cancel",
+    "save-btn": "Save",
+    "sin": "Social Insurance Number (SIN)",
+    "year-of-birth": "Year of birth",
+    "confirm-checkbox": "I confirm that my spouse or common-law partner is aware and has agreed to share their personal information.",
+    "help-message": {
+      "sin": "Enter the 9-digit SIN"
+    },
+    "error-message": {
+      "marital-status-required": "Select marital status",
+      "confirm-required": "Checkbox must be selected",
```

This error message was flagged as an accessibility issue for renewals and should be changed for apply too:
```suggestion
      "confirm-required": "Consent from spouse or common-law partner is required to proceed",
```